docs/setup/permissions.md

```diff
@@ -15,36 +18,55 @@ with careful consideration for both battery life and security - only opening con
 
 #### External Storage Modification
 
-Attachments and settings can be loaded and saved onto external storage. In order read and write these files we need 
+### Read your contacts
+
+In order to mark e-mail that comes from your contacts and use your contacts to provide email address auto-completion, we
+need access to your contacts.
+
+### Modify your contacts
+
+In order to note that a contact has been emailed, we need write-access.
```

Comment:
We no longer request the permission to write contacts. For the reason see https://developer.android.com/guide/topics/providers/contacts-provider#ObsoleteData